card-style.cls

```diff
@@ -76,7 +77,7 @@
       \draw [color=\cardStyle@borderColor,rounded corners=4mm,line width=4mm]
         (current page.north east)
         +(-24mm,0mm) coordinate(a)
-        +(0mm, -3mm) coordinate(b)
+        +(0mm, -2.5mm) coordinate(b)
```

Comment:
This is chosen so that the border is half way between item type and reference on the upper right corner of the cards.